tests/compiler/function.ts

```diff
@@ -1,34 +1,40 @@
 function v(): void {}
+function V(): void { return; }
```

Comment:
Signatures here seem to use the old naming scheme. `V` would be `v128` now, and the others changed as well. Might make sense to update. New one is here: https://github.com/AssemblyScript/assemblyscript/blob/master/src/types.ts#L369

Reply:
Done!